decoder/decoder.go

```diff
@@ -28,6 +28,8 @@ type Decoder struct {
 	utmMedium string
 	// utm_content parameter, e.g. documentHover or documentLink
 	useUtmContent bool
+
+	PrefillRequiredAttributes bool
```

Comment:
```suggestion
	// PrefillRequiredFields enriches label-based completion candidates with required attributes and blocks
	PrefillRequiredFields bool
```
nitpick: "attributes -> fields" is a minor name change, but I think it better reflects that we're completing both attributes and blocks.